### PR TITLE
Extract font picker from Bloom for stand-alone target

### PR DESCRIPTION
**Pull request by imnasnainaec**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,8 +12,13 @@
     "test": "nx run-many --all -t test"
   },
   "dependencies": {
+    "@emotion/react": "^11.13.0",
+    "@emotion/styled": "^11.13.0",
+    "@mui/icons-material": "^5.16.5",
+    "@mui/material": "^5.16.5",
     "react": "18.3.1",
-    "react-dom": "18.3.1"
+    "react-dom": "18.3.1",
```
Comment (irahopkinson) [inline package.json:20]: BTW I thought for React these need to be `peerDependencies` so you don't get 2 copies of React running where you are using these components - that would mean moving these to `devDependencies` and adding them with suitable ">=" to `peerDependencies`. Maybe your build system has a way around this.

File: libs/bloom-lfp/src/BloomFontPicker.tsx [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+  languageNumber: number;
+  onChangeFont?: (fontname: string) => void;
+  /** Use this if you need to modify the style of popup menus by increasing z-index
+   * (e.g., to make the popup be in front of the font dialog) */
+  popoverZindex?: string;
+  suitabilityCheck?: boolean;
+  useL10n?: useL10nHookType;
+}
+
+const FontSelectComponent: FunctionComponent<FontSelectProps> = (props) => {
```
Comment (irahopkinson) [inline libs/bloom-lfp/src/BloomFontPicker.tsx]: BTW I believe React is recommending regular functions over arrow functions for components these days.

File: README.md
Comment (irahopkinson) [inline README.md:1]: Perhaps add `bloom-lfp` in as a project after L49

File: libs/bloom-lfp/src/WinFormsStyleSelect.tsx [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+// This seemed to be the only way to affect the css of the popped up list, since it's a completely
+// separate html element from this component.
+// TODO: migrate away from MUIv4's useStyle usage:
+/*const useStyles = makeStyles(() => ({
+  menuPaper: {
+    maxHeight: 225,
+    padding: 3,
+  },
+}));*/
```
Comment (irahopkinson) [inline libs/bloom-lfp/src/WinFormsStyleSelect.tsx:26]: Can you delete this code now since its not doing anything?

**Review by irahopkinson — APPROVED** [review]
👍 LGTM

I don't mind doing these reviews but I'm not sure that I'm contributing much value anymore now that things are setup and you look like you have more React experience than I do.

File: package.json (hunk 1 above)
Comment (imnasnainaec) [reply package.json:20]: Ah, I had forgotten about peerDependencies, thank you! That'll be a next pr.

File: libs/bloom-lfp/src/WinFormsStyleSelect.tsx (hunk 3 above)
Comment (imnasnainaec) [reply libs/bloom-lfp/src/WinFormsStyleSelect.tsx:26]: There's style info in there I may still need to add elsewhere.

**Pull request merged**